### PR TITLE
New file for  pgr_contracted_blob_h structure

### PR DESCRIPTION
**Pull request by daas-ankur-shukla**
Fixes [#10](https://github.com/daas-ankur-shukla/pgrouting/issues/10)

Changes proposed in this pull request:
- Commented out pgr_contracted_blob structure in include/pgr_types.h
- Created new file pgr_contracted_blob.h
- Edited inclusion of header files in contractGraph.c and contractGraph_Driver.h

@pgRouting/admins

### CODE REVIEW

**Comment by daas-ankur-shukla** [comment]
[last commit](https://github.com/pgRouting/pgrouting/pull/793/commits/89a4d4595ec324123d5f77333ca945e634ebf3b2) solved [#11](https://github.com/daas-ankur-shukla/pgrouting/issues/11)

File: include/c_types/contracted_rt.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/*PGR-GNU*****************************************************************
+File: pgr_types.h
```
Comment (cvvergara) [inline include/c_types/contracted_rt.h]: File name is: contracted_rt.h

File: include/c_types/contracted_rt.h [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Copyright (c) 2015 Celia Virginia Vergara Castillo
+Mail: vicky_vergara@hotmail.com
```
Comment (cvvergara) [inline include/c_types/contracted_rt.h]: Ok, the design is by me, but the change is by you so lets make this:

Copyright (c) 2017 pgRouting developers
Mail: project@pgrouting.org

File: include/c_types/contracted_rt.h [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+ * ***********************************************************************/
+typedef struct {
+    int64_t id;
+    char* type;
+    int64_t source;
+    int64_t target;
+    double cost;
+    int64_t *contracted_vertices;
+    int contracted_vertices_size;
+} pgr_contracted_blob;
```
Comment (cvvergara) [inline include/c_types/contracted_rt.h]: This needs to be:
Contracted_rt

**Comment by daas-ankur-shukla** [comment]
@cvvergara i have added the proposed changes. kindly review

**Pull request merged**